setup.py

```diff
@@ -40,10 +40,10 @@
             "sqlalchemy_utils==0.38.2",
             "PyMySQL==1.0.2",
         ],
-        "redis": ["redis==4.1.1"],
+        "redis": ["redis==4.2.2"],
```

Comment:
We had problems in onboarding service because `fakeredis` was requiring a redis version minor than `4.2.0` (see [source](https://github.com/jamesls/fakeredis/blob/0303b2ddfa4d6cd5098a0a29b23d300296307c59/setup.cfg#L29)) but I dont know if it was problem of resolve deps due the old pip version that we use there

Reply:
Which version of fakeredis are you using? (https://github.com/jamesls/fakeredis/tree/0303b2ddfa4d6cd5098a0a29b23d300296307c59#171) 

Maybe we could wait for new versions of fakeredis 🤔

Reply:
Yes, I think it could give problems trying to resolve the dependencies with latest versions of pip

Reply:
It seems that the new versions of fakeradis now support the new versions of redis: https://github.com/cunla/fakeredis-py/releases

There are also new versions of redis client: https://github.com/redis/redis-py/releases

Reply:
👍🏼

Reply:
cool!